src/store/traction/pacbio/pools/getters.js

```diff
@@ -1,16 +1,32 @@
+const buildRunSuitabilityErrors = ({ pool, libraries }) => [
+  ...pool.run_suitability.errors.map(({ detail }) => `Pool ${detail}`),
+  ...libraries.flatMap((library) => {
+    const libraryName = `Library ${library.id} (${library.sample_name})`
+    return library.run_suitability.errors.map(({ detail }) => `${libraryName} ${detail}`)
+  }),
+]
+
```

Comment:
I considered making this more robust, to handle the missing property. I stopped short, as it added a bit of complexity, for a situation that should only really happen if we deploy the UI changes before service.

However, I'm not 100% convinced, so happy for second opinions.